### ISSUE TITLE
FAQ translation

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo.github.io#152]
I want to start a new approach of asking the community for help in translating the FAQ.

For this reason I started a small Wiki page describing some basics (for people not used to Github) about how they can contribute.

Maybe someone (@SammysHP ?) can review the wiki page and make additions/corrections where needed.

https://github.com/cgeo/cgeo.github.io/wiki/How-to-contribute-translations-for-the-c:geo-FAQ

**Comment by kumy** [comment]
Wouldn't be easier for translators to use the crowdin platform?
I think they can handle .yml files.

Le 2017-08-17 16:01, Lars a écrit :

**Comment by SammysHP** [comment]
Also we should probably add one file for each language (as suggested earlier in my second draft).

**Comment by Lineflyer** [comment]
We have issues for both topics AFAIR. But as none of these is true until now,  I wanted to go another way. But if one of you is interested to bring it to crowdin, that would be even better.

**Comment by kumy** [comment]
Having separated files by languages would help a bit.

**Comment by Lineflyer** [comment]
@shroudedcode Are you maybe interested to help with making the hompage "compatible" with using a translation service (i.e. Crowdin) in our case? Some ideas are mentioned above already.

I think it would help a lot to have the whole page (or at least the FAQ) localized in more languages to allow users to help themselves in case of questions/problems.

**Comment by shroudedcode** [comment]
@Lineflyer If we were using a more flexible static site generator/web server solution, I'd say yes. But I really don't want to do this with Jekyll, because I would probably have to implement it in a kind of hacky way which is often the case with Jekyll.

[`electronjs.org`](https://github.com/electron/electronjs.org/) which [previously used Jekyll as well](https://github.com/electron/electronjs.org/tree/gh-pages) switched to a custom Node.js solution because of this which allowed them to implement [an great system for localization](https://github.com/electron/electron-i18n). I also use Node.js to build [my personal website](https://github.com/shroudedcode/website) and it's just great.

In other words: I'd rather rebuild this site from scratch using a custom Node.js build process than having to work around Jekyll's limitations or wait for certain plugins to be allowed on GH pages.
